src/website/app/demos/Dialog/examples/Dialog/trigger.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+import { Component } from 'react';
+import Button from '../../../../../../library/Button';
+import Dialog from '../../../../../../library/Dialog';
+
+export default {
+  id: 'trigger',
+  title: 'Trigger',
+  description: `Toggle the display of a Dialog with the \`isOpen\` prop.`,
```

Comment:
I think this is a more appropriate description for this example than what we had, but then to me, it makes more sense to have this be the 2nd example instead of the first.

Reply:
I debated that myself, but I think you just convinced me.